tests/migrations/test_writer.py

```diff
@@ -537,8 +537,9 @@ def test_serialize_datetime(self):
                 self.assertSerializedResultEqual(
                     datetime.datetime(2012, 1, 1, 1, 1, tzinfo=tzinfo),
                     (
-                        "datetime.datetime(2012, 1, 1, 1, 1, tzinfo=utc)",
-                        {"import datetime", "from django.utils.timezone import utc"},
+                        "datetime.datetime(2012, 1, 1, 1, 1, "
+                        "tzinfo=datetime.timezone.utc)",
```

Comment:
Single-lined this is 92 chars, so alternative would be: 

```suggestion
                        "datetime.datetime(2012, 1, 1, 1, 1, tzinfo=datetime.timezone.utc)",  # noqa
```